cs3/gateway/v1beta1/gateway_api.proto

```diff
@@ -169,6 +170,29 @@ service GatewayAPI {
   // Unsets arbitrary metdata into a storage resource.
   // Arbitrary metadata is returned in a cs3.storage.provider.v1beta1.ResourceInfo.
   rpc UnsetArbitraryMetadata(cs3.storage.provider.v1beta1.UnsetArbitraryMetadataRequest) returns (cs3.storage.provider.v1beta1.UnsetArbitraryMetadataResponse);
+  // Locks a storage resource.
```

Comment:
@redblom I fear something went wrong with your merge (shouldn't have it been a rebase?), there seem to be duplicated code in the changes - see e.g. this block that comes from a previously merged PR. Can you please have a look?